### PR TITLE
📖 update release notes for v0.3.0

### PR DESCRIPTION
**Pull request by jasonliu747**
Signed-off-by: Jason Liu <jasonliu747@gmail.com>

### CODE REVIEW

File: blog/2022-05-07-release/index.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+| cfsQuotaBurstPeriodSeconds | int       | Default value: `-1`. Unit: seconds. This indicates that the time period in which the container can run with an increased CFS quota is unlimited. This field specifies the time period in which the container can run with an increased CFS quota, which cannot exceed the upper limit specified by `cfsQuotaBurstPercent`.                                                                                                              |
+| sharePoolThresholdPercent  | int       | Default value: `50`. Unit: %. This field specifies the CPU utilization threshold of the node. If the CPU utilization of the node exceeds the threshold, the value of cfs_quota in cgroup parameters is reset to the original value.                                                                                                                                                                                                     |
+
+## L3 cache and MBA resource isolation
+
+Tasks of different priorities are usually deployed on the same machine. This may cause tasks to compete for computing
```
Comment (hormes) [inline blog/2022-05-07-release/index.md:132]: use pods, do not introduce other concepts to cause ambiguity

Comment (jasonliu747) [reply blog/2022-05-07-release/index.md]: fixed

File: blog/2022-05-07-release/index.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+| cfsQuotaBurstPeriodSeconds | int       | Default value: `-1`. Unit: seconds. This indicates that the time period in which the container can run with an increased CFS quota is unlimited. This field specifies the time period in which the container can run with an increased CFS quota, which cannot exceed the upper limit specified by `cfsQuotaBurstPercent`.                                                                                                              |
+| sharePoolThresholdPercent  | int       | Default value: `50`. Unit: %. This field specifies the CPU utilization threshold of the node. If the CPU utilization of the node exceeds the threshold, the value of cfs_quota in cgroup parameters is reset to the original value.                                                                                                                                                                                                     |
+
+## L3 cache and MBA resource isolation
+
+Pods of different priorities are usually deployed on the same machine. This may cause tasks to compete for computing
```
Comment (hormes) [inline blog/2022-05-07-release/index.md:132]: use pods, avoid introducing other concepts to cause ambiguity

File: blog/2022-05-07-release/index.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+- The memory limit of the container. If the amount of memory that a container uses, including the page cache, is about
+  to reach the memory limit of the container, the memory reclaim mechanism of the OS kernel is triggered. As a result,
+  the application in the container may not be able to request or release memory resources as normal.
+- The memory limit of the node. If the memory limit of a container is greater than the memory request of the container,
+  the container can overcommit memory resources. In this case, the available memory on the node may become insufficient.
+  This causes the OS kernel to reclaim memory from containers. As a result, the performance of your application is
+  downgraded. In extreme cases, the node cannot run as normal.
+
+To improve the performance of applications and the stability of nodes, Koordinator provides the memory QoS feature for
+containers. **To use this feature, you must use Anolis OS as the node OS.** After
```
Comment (hormes) [inline blog/2022-05-07-release/index.md:193]: Recommended, not mandatory. On other OS, we will try our best to adapt, and users can enable it without side effects.

**Pull request merged**